man/sd_uid_get_state.xml

```diff
@@ -126,16 +133,27 @@
     of the "primary" session of a user. If the user has graphical
     sessions, it will be the oldest graphical session. Otherwise, it
     will be the oldest open session.</para>
+
+    <para><function>sd_uid_get_login_time()</function> may be used to
+    determine the time the user's service manager has been invoked,
```

Comment:
The service manager part also applies to lingering users, so we should be more precise 🤔

Reply:
That's explained in the next line which you did not quote: "user's first active session"
According to logind documentation lingering is not an active user session, only running processes.